### DISCUSSION TITLE
Duda sobre Motivación y Conclusión en el informe

### DISCUSSION THREAD

**Discussion by gabo1243** [IIC2133-PUC/2024-1#26] [Tarea 0] [answered]
Hola, en el template del informe aparecen apartados de motivación y conclusión, ¿Qué se espera que uno escriba en cada uno? porque no se qué tanto puedo concluir y qué es lo que me motiva a resolver la tarea más allá de representar problemas en algoritmos. Por otra parte ¿Son considerados en las 2 páginas?

Gracias de antemano.

**Comment by benjavicente** [comment, marked as answer]
Motivación es el _¿para qué?_: que es lo relevante de lo que se está haciendo. Conclusión sería resumir lo explicado en el informe, relacionándolo con la motivación. Ambos pueden ser bien breves, y son considerados en las 2 páginas.
